cache/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
```

Comment:
Ah, fair enough if you have a reason :) Yes, git doesn't deal with directories at all, only files.